### PR TITLE
fix(android): Remove extra activity

### PR DESCRIPTION
**Pull request by hiteshjain29**
### Changelog:
- Remove extra activity(view) layer from android.
- Added fallback response for passive forms.

### CODE REVIEW

File: plugin.xml [hunk 1]
```diff
@@ -41,7 +41,8 @@
     <dependency id="cordova-plugin-add-swift-support" version="2.0.2">
       <preference name="UseSwiftLanguageVersion" value="4.0" />
     </dependency>
+     <pods-config ios-min-version="9.0" use-frameworks="true"/>
     <source-file src="src/ios/UsabillaCordova.swift" />
-    <framework src="Usabilla" type="podspec" spec="~>6.7" />
+    <pod name="Usabilla" spec="~>6.4.7"/>
```
Comment (kioli) [inline plugin.xml:46]: Is this part of this commit (remove android activity) or should it be in a commit apart?

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1,5 @@
+# Mac
```
Comment (kioli) [inline .gitignore:1]: This should probably be in a separate commit titled `update .gitignore` file

File: scripts/android/addResourcesClassImport.js [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+    console.log('Found plugin sources root', pluginSourcesRoot);
+
+    glob(pluginSourcesRoot + '/**/*.java', { absolute: true }, function (err, files) {
+        if (err) {
+            console.error('Error when reading file:', err);
+            deferral.reject();
+            return;
+        }
+
+        //files.filter(function (file) { return path.extname(file) === '.java'; })
```
Comment (kioli) [inline scripts/android/addResourcesClassImport.js:55]: Can this commented line be removed?

File: scripts/android/addResourcesClassImport.js [hunk 4]
```diff
@@ -0,0 +1,103 @@
...
+                        }
+                    }
+                });
+
+                deferral.resolve();
+            });
+    });
+
+    return deferral.promise;
+}
```
Comment (kioli) [inline scripts/android/addResourcesClassImport.js:103]: EOF line needed

File: src/android/UsabillaCordova.java [hunk 5]
```diff
@@ -8,55 +8,85 @@
...
+        public void onReceive(Context context, Intent intent) {
+            if (intent != null) {
+                final JSONObject result = prepareResult(intent, FeedbackResult.INTENT_FEEDBACK_RESULT);
+                callbackContext.success(result);
+            }
 
-    private CallbackContext callbackContext;
+            final Activity activity = ((MainActivity) cordova.getActivity());
+            if (activity instanceof FragmentActivity) {
+                FragmentManager supportFragmentManager = ((FragmentActivity) activity).getSupportFragmentManager();
```
Comment (kioli) [inline src/android/UsabillaCordova.java:71]: the FragmentManager can be pre-pended with `final`

File: src/android/UsabillaCordova.java [hunk 6]
```diff
@@ -8,55 +8,85 @@
...
+                Fragment fragment = supportFragmentManager.findFragmentByTag(FRAGMENT_TAG);
```
Comment (kioli) [inline src/android/UsabillaCordova.java:72]: the Fragment can be pre-pended with `final`

File: src/android/activity/MainActivity.java [hunk 7]
```diff
@@ -1,59 +1,48 @@
...
+import io.cordova.hellocordova.R;
 import uk.co.reallysmall.cordova.plugin.fragment.CordovaFragment;
 
 public class MainActivity extends AppCompatActivity {
 
-    private IntentFilter closeCampaignFilter = new IntentFilter(UbConstants.INTENT_CLOSE_CAMPAIGN);
-    private BroadcastReceiver receiverCampaignClosed;
-
     public CordovaFragment currentFragment;
+    public FakeR fakeR;
```
Comment (kioli) [inline src/android/activity/MainActivity.java:18]: Why have we extracted it? Do we use it anywhere else than in the `OnCreate`

File: src/ios/UsabillaCordova.swift [hunk 8]
```diff
@@ -135,10 +135,11 @@ class UsabillaCordova: CDVPlugin {
         )
     }
     
-    func error() {
+    func error(error: Any) {
```
Comment (kioli) [inline src/ios/UsabillaCordova.swift:138]: This is a swift file, it may also use its own commit

File: src/android/activity/MainActivity.java (hunk 7 above)
Comment (hiteshjain29) [reply src/android/activity/MainActivity.java:18]: nope we only use it on on create

Comment (kioli) [reply src/android/activity/MainActivity.java:18]: So we could move it in there to be a local field

Comment (hiteshjain29) [reply src/android/activity/MainActivity.java]: done

**Comment by hiteshjain29** [comment]
> I think convention for commits that do not change any version (major, minor nor patch) is to start with `chore(xxx):` where xxx is the topic the commit is regarding

Changed, Done
